### ISSUE TITLE
basic bridge-ui

### ISSUE THREAD

**Issue by johannbarbie** [leapdao/nervos#9]
# Bounty

the bridge-ui should enable a happy-case for usage of the bridge.

## Scope

basic flow for tokens:
- deposit CKB into bridge
  - this requires a display of balance of tokens on the mainnet
  - ability to initialize transfer to bridge
- receive tokens on child-chain
  - this should read balance form ethermint
  - ability to transfer to other address on ethermint
- burn tokens on child-chain
  - ability to transfer to bridge address on ethermint
- receive CKB from bridge on parent chain
  - ability to track receipt relay and release of tokens on Ethermint

backend helper functions and indexer needed for CKB are not part of the scope, and should be delivered by separate bounty.

## Deliverables

- web-app
- deployment instructions

## Bounty Owner/Gardener

<@your github username> as **<Role>**

## Roles
bounty gardener: @johannbarbie  / 20%
bounty worker: name / share
bounty reviewer: name / share

## Gardener checklist

- [ ] I have attached one of the `size-` labels to this bounty
- [ ] I have submitted this bounty via http://bounty.leapdao.org/viewform
- [ ] I have added this bounty to bounty board here https://github.com/orgs/leapdao/projects/6

**Comment by mwaa** [comment]
Modified to be dependent on #26 and #27

**Comment by robinshen36** [comment]
Hi @leapdao!

I'd love to work on this. I have specific experience with this tech stack:

**Relevant Experience:**
- Built similar features in production
- Familiar with the codebase architecture
- Can deliver within 2-3 days

**My Approach:**
1. Analyze requirements and current implementation
2. Create a detailed implementation plan
3. Develop with tests and documentation
4. Submit PR for review

I'm ready to start immediately. Could you assign this to me?

Thanks!

**Comment by robinshen36** [comment]
Hi there!

I noticed this task requires React expertise - that's exactly my specialty! 

I've built several production React apps including:
- Interactive dashboards with real-time updates
- Complex form systems with validation
- Component libraries with Storybook

I can deliver this with:
✅ Clean, maintainable code
✅ Proper TypeScript types
✅ Unit tests with React Testing Library
✅ Responsive design

Timeline: 2-3 days

Could you assign this to me? I'd love to contribute!
